doc/faq/README.md

```diff
@@ -2,7 +2,7 @@
 
 ### Which versions of Cassandra does the driver support?
 
-The latest version of the driver supports any Cassandra version greater than 1.2.
+The latest version of the driver supports any Cassandra version starting with 1.2.
```

Comment:
In our compatibility matrix we don't list C* 1.2 or 2.0 any more since they are considered EOL'd, so maybe we should just change this to 2.1+, even if C* 1.2 happens to work we aren't testing against it specifically this release.